src/creatwth.cpp

```diff
@@ -1019,6 +1019,11 @@ VOID CALLBACK DetourFinishHelperProcess(_In_ HWND,
         rlpDlls = NULL;
     }
 
+    // Delete the payload after execution to release the memory occupied by it
+    if (s_pHelper != NULL) {
+        DetourFreePayload(s_pHelper);
```

Comment:
We should set s_pHelper to NULL here?

Reply:
Good idea, the latest submission has been changed in accordance with your suggestions.